scripts/stubsabot.py

```diff
@@ -498,6 +498,7 @@ async def determine_action(stub_path: Path, session: aiohttp.ClientSession) -> U
         "Release": f"{pypi_info.pypi_root}/{relevant_version}",
         "Homepage": project_urls.get("Homepage"),
         "Repository": stub_info.upstream_repository,
+        "Typeshed stubs": f"https://github.com/{TYPESHED_OWNER}/typeshed/tree/main/{stub_info.distribution}",
```

Comment:
Was this stubsabot change meant to be in this PR ?

Reply:
Yeah, though I'm happy to move it to another PR if you prefer! It's basically just the same thing as one of the changes I also made to `stubtest_third_party.py` in this PR (it allows us to click straight through to the source for the typeshed stubs from a stubsabot PR, rather than having to manually navigate to them

Reply:
I'm ok with it, it's a simple PR. 🙂 Just wanted to make sure it was intended.